treeherder/perf/migrations/0012_rename_summary_last_updated.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-02-20 15:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('perf', '0011_inc_extra_options_length'),
+    ]
+
+    operations = [
+        migrations.RenameField(
```

Comment:
Could you use the `./manage.py sqlmigrate` command to output the SQL that will be run by this migration to check Django is doing the right thing? (Occasionally it generates a suboptimal set of commands)